### PR TITLE
Acceptance test for Clique when configured to not create empty blocks

### PR DESCRIPTION
**Pull request by fab-10**

### CODE REVIEW

**Review by Gabriel-Trintinalia — APPROVED** [review]
lgtm

**Review by jframe — APPROVED** [review]
LGTM

File: acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/clique/CliqueMiningAcceptanceTest.java [hunk 1]
```diff
@@ -42,6 +42,14 @@ public void shouldMineTransactionsOnSingleNode() throws IOException {
     cluster.verify(receiver.balanceEquals(3));
   }
 
+  @Test
+  public void shouldNotMineBlocksIfNoTransactionsOnSingleNode() throws IOException {
+    final BesuNode minerNode = besu.createCliqueNoEmptyBlockNode("miner1");
+    cluster.start(minerNode);
+
+    cluster.verify(clique.noNewBlockCreated(minerNode));
```
Comment (jframe) [inline acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/clique/CliqueMiningAcceptanceTest.java:50]: Would also be good to have a test showing that when we have transactions again that we start mining blocks again

Comment (fab-10) [reply acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/clique/CliqueMiningAcceptanceTest.java:50]: godd point, will add this test too

Comment (fab-10) [reply acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/clique/CliqueMiningAcceptanceTest.java:54]: @jframe test added, please re-review

**Review by jframe — APPROVED** [review]
LGTM

**Pull request merged**